source/includes/k8s/operator-values.yaml

```diff
@@ -32,14 +28,14 @@ operator:
   ###
   # Specify the Operator container image to use for the deployment.
   # ``image.tag``
-  # For example, the following sets the image to the ``quay.io/minio/operator`` repo and the v6.0.3 tag.
+  # For example, the following sets the image to the ``quay.io/minio/operator`` repo and the v7.0.0 tag.
```

Comment:
I guess our automation can't directly replace a version number in files that are included this way?

Reply:
We could potentially grep for it, but https://github.com/minio/operator/blob/master/helm/operator/values.yaml ultimately dictates the content. Ideally we would only update it there so we have minimal skew.